sos/report/plugins/storcli.py

```diff
@@ -0,0 +1,29 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+
+# This sosreport plugin is meant for sas adapters.
+# This plugin logs inforamtion on each adapter it finds.
+
+from sos.report.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+
+
+class SAS3ircu(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+
+    short_desc = 'storcli PCI adapter information'
+
+    plugin_name = "storcli"
+    commands = ("storcli",)
+
+    def setup(self):
+
+        # Display list of adapters
+        self.collect_cmd_output("storcli show all", timeout=5)
+        self.collect_cmd_output("storcli /cx[/eall]/sall show", timeout=5)
+
```

Comment:
You can use a single `collect_cmd_output()` call here since all the other arguments are the same - just pass both commands as a list.

Do you have example output for the 2nd command (or the whole plugin)?

Reply:
Thanks for the review..sure will make changes and send new  version of patchset

Reply:
Yes as you mentioned we have to rework on this patch
please find output of the second commands

storcli /c0/eall/sall show
CLI Version = 007.1316.0000.0000 Mar 12, 2020
Operating system = Linux 4.14.0-115.el7a.ppc64le
Controller = 0
Status = Success
Description = Show Drive Information Succeeded.
Drive Information :
=================
-----------------------------------------------------------------------------------------------------
EID:Slt DID State DG       Size Intf Med SED PI SeSz Model                                   Sp Type
-----------------------------------------------------------------------------------------------------
252:0     8 JBOD  -  223.570 GB SATA SSD N   N  512B MTFDDAK240TCB        01EJ260 01EJ260IBM U  -
252:2    10 JBOD  -  223.570 GB SATA SSD N   N  512B MTFDDAK240TCB        01EJ260 01EJ260IBM U  -
252:3    11 JBOD  -  223.570 GB SATA SSD N   N  512B MTFDDAK240TCB        01EJ260 01EJ260IBM U  -
252:4    15 UBad  F    2.182 TB SAS  HDD N   Y  4 KB ST2400MM0129                            U  -
252:5    14 Onln  0    2.182 TB SAS  HDD N   Y  4 KB ST2400MM0129                            U  -
252:6    13 Onln  1    2.182 TB SAS  HDD N   N  4 KB ST2400MM0129                            U  -
252:7    20 JBOD  -    2.182 TB SAS  HDD N   Y  4 KB ST2400MM0129                            U  -
-----------------------------------------------------------------------------------------------------

Reply:
I will rework and send this patch later. right now I will update all other comments and send new version of patchset except storcli